src/GitVersionTfsTask/Tasks/GitVersion/GitVersion.ts

```diff
@@ -0,0 +1,99 @@
+import tl = require('vsts-task-lib/task');
+import path = require('path');
+import fs = require('fs-extra');
+import q = require('q');
```

Comment:
I've had some issue with q not being properly loaded (vsts task lib relies on it) in the past. I'll see if it can be removed.